### PR TITLE
[UBER-73] Remove project default time report day

### PR DESCRIPTION
**Pull request by s0gorelkov**
# Contribution checklist

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [x] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [x] - Does the translations are up to date?
* [x] - Does it well tested?
* [x] - Tested for Chrome.
* [ ] - Tested for Safari.
* [x] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [x] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [x] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

[UBER-73](https://front.hc.engineering/workbench/platform/tracker/UBER-73)

### CODE REVIEW

File: models/tracker/src/migration.ts [hunk 1]
```diff
@@ -151,16 +149,13 @@ async function fixProjectsIssueStatusesOrder (tx: TxOperations): Promise<void> {
...
-  const projects = await tx.findAll(tracker.class.Project, {
-    defaultTimeReportDay: { $exists: false }
-  })
-  await Promise.all(
-    projects.map((project) =>
-      tx.update(project, {
-        defaultTimeReportDay: TimeReportDayType.PreviousWorkDay
-      })
-    )
+async function removeProjectDefaultTimeReportDay (client: MigrationClient): Promise<void> {
```
Comment (haiodo) [inline models/tracker/src/migration.ts]: Let's just don't do anything.

Comment (s0gorelkov) [reply models/tracker/src/migration.ts]: I think we need apply migrations after updating the "Project" model, don't we?

Comment (haiodo) [reply models/tracker/src/migration.ts]: We do not need to cleanup from existing projects.

Comment (s0gorelkov) [reply models/tracker/src/migration.ts]: Done.

**Review by haiodo — APPROVED** [review]

**Pull request merged**